### ISSUE TITLE
Complete Simple JS Input Form.

### ISSUE THREAD

**Issue by mpekim** [mpekim/hello-world#9]
PHP isn't necessarily required to create input for the website. You can make a simple input form using:
- HTML Forms
- JS input methods (*prompt()*, *.value*, and other methods)

As of right now, the website can accept a text string or hexidecimal value to change the color of the screen.

**Comment by mpekim** [comment]
**4/20:** Consider how the data will be used if PHP is not needed. Will they stay within the script? Will they be transferred to files?

**Comment by mpekim** [comment]
**5/22:** At this time, prompt accepts a hex value or string to change the page background color.

**Comment by mpekim** [comment]
**5/23:** Applied for a Junior PHP position.

**Comment by mpekim** [comment]
**6/1:** Waiting on job application replies.

**Comment by mpekim** [comment]
**6/12:** Job interview went well, and now references have been used. It is likely that I will acquire this position.

**Comment by mpekim** [comment]
**6/22:** Planning on working now that I have Wi-Fi.

**Comment by mpekim** [comment]
**6/29:** Day off, no progress on issue today.

**Comment by mpekim** [comment]
**7/13:** Still working on input for code snippets. Make sure that errors and warnings are caught later on.

**Comment by mpekim** [comment]
**8/14:** Errors can be found by finding improper strings of code.

**Comment by mpekim** [comment]
**10/26:** In order to close issue, we will need to make sure that the input form can properly read lines of JS code.

**Comment by mpekim** [comment]
**11/29:** Currently coding and testing simple JS input. As of right now, only single lines of code work.

**Comment by mpekim** [comment]
**12/5:** Once single-line JS input works, consider new issue for functional language selection.

**Comment by mpekim** [comment]
**12/15:** Currently focusing on remote work.

**Comment by mpekim** [comment]
**12/25:** Merry Christmas!

**Comment by mpekim** [comment]
**1/1:** Happy New Year!

**Comment by mpekim** [comment]
**1/12:** Trying to figure out more work for this issue.

**Comment by mpekim** [comment]
**1/19:** creating JS code for dynamic features on remote work site.

**Comment by mpekim** [comment]
**1/30:** Making next major push to remote work (based on this issue).

**Comment by mpekim** [comment]
**2/8:** Cashed out invoice. (Remote work)

**Comment by mpekim** [comment]
**2/19:** Catch up on remote work.

**Comment by mpekim** [comment]
**3/4:** Sent out applications for job opportunities.

**Comment by mpekim** [comment]
**3/16:** Continued work on mobile CSS for ravage site.

**Comment by mpekim** [comment]
**3/22:** Continuing mobile CSS work for Ravage.

**Comment by mpekim** [comment]
**3/27:** Got new assignments for remote work.

**Comment by mpekim** [comment]
**4/4:** Check status of issue; see if it can be closed.
